.github/CODEOWNERS

```
@@ -34,17 +34,7 @@ pnpm-lock.yaml
 # crates in alphabetical order
 /crates @vercel/web-tooling
 # overrides for crates that are owned by turbo-oss
-/crates/globwatch @vercel/turbo-oss
-/crates/pidlock @vercel/turbo-oss
-/crates/turbopath @vercel/turbo-oss
-/crates/turborepo @vercel/turbo-oss
-/crates/turborepo-api-client @vercel/turbo-oss
-/crates/turborepo-ffi @vercel/turbo-oss
-/crates/turborepo-lib @vercel/turbo-oss
-/crates/turborepo-lockfiles @vercel/turbo-oss
-/crates/turborepo-scm @vercel/turbo-oss
-/crates/turborepo-vercel-api-mock @vercel/turbo-oss
-/crates/turbo-updater @vercel/turbo-oss
+/crates/turborepo-* @vercel/turbo-oss
```

```suggestion
/crates/turborepo* @vercel/turbo-oss
```
Otherwise it doesn't include turborepo itself.